admin_manual/installation/system_requirements.rst

```diff
@@ -12,31 +12,33 @@ Nextcloud needs a minimum of 128MB RAM, and we recommend a minimum of 512MB.
 Recommended setup for running Nextcloud
 ---------------------------------------
 
-For best performance, stability, support, and full functionality we recommend:
+For best performance, stability, and functionality, recommendations are:
 
 * Red Hat Enterprise Linux 7 / Ubuntu 16.04 LTS
 * MySQL/MariaDB
-* PHP 7.0, 7.1, or 7.2
+* PHP 7.0, 7.1 or 7.2.
 * Apache 2.4 with mod_php
 
-Supported platforms
--------------------
-
-* Server: Linux (Debian 7, SUSE Linux Enterprise Server 11 SP3 & 12, 
-  Red Hat Enterprise Linux/CentOS 6.5 and 7 (7 is 64-bit only), Ubuntu 14.04 LTS, 16.04 LTS)
-* Web server: Apache 2 (mod_php, php-fpm) or Nginx (php-fpm) 
-* Databases: MySQL/MariaDB 5.5+; PostgreSQL 9, 10; Oracle 11g (currently only possible 
-  if you `contact us <https://nextcloud.com/enterprise>` as part of a subscription)
-* PHP 5.6 + required (up to PHP 7.2)
-* Hypervisors: Hyper-V, VMware ESX, Xen, KVM
-* Desktop: Windows XP SP3 (EoL Q2 2015), Windows 7+, macOS 10.10+ (64-bit 
-  only), Linux (CentOS 6.5, 7 (7 is 64-bit only), Ubuntu 12.04 LTS, 14.04 LTS, 
-  14.10, Fedora 20, 21, openSUSE 12.3, 13, Debian 7 & 8).
-* Mobile apps: iOS 7+, Android 4+
-* Web browser: IE11+, Microsoft Edge, Firefox 14+, Chrome 18+, Safari 7+
+For a Nextcloud installation with efficient and reliable scaling over ~100 users, see the `deployment recommendations <https://portal.nextcloud.com/article/nextcloud-deployment-recommendations-7.html>`
+
+Working platforms
+-----------------
+
+* Server: GNU/Linux (Debian 8 (Jessie), 9 (Stretch), SUSE Linux Enterprise Server 11 SP3 & 12, openSUSE LEAP 42.1 or newer,  Red Hat Enterprise Linux/CentOS 6.5 and 7 (7 is 64-bit only), Ubuntu 14.04 (Trusty) LTS, 16.04 (Xenial) LTS, 18.04 (Bionic) LTS)
+* Web server: Apache 2.4 (mod_php, php-fpm) or Nginx (php-fpm) 
+* Databases: MySQL or MariaDB 5.5+; PostgreSQL 9, 10; Oracle 11g (currently only possible 
+  as part of an `enterprise subscription <https://nextcloud.com/enterprise>`) 
+* PHP 7.0, 7.1 or 7.2 required
```

Comment:
PHP 5.6 should be included as well. `master` is the one that we use for the 13 release as of now.

Reply:
Fixed. Awaiting :+1:  :smile: